orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/IgorService.java

```diff
@@ -85,4 +85,19 @@ GoogleCloudBuild runGoogleCloudBuildTrigger(
       @Path("account") String account,
       @Path("triggerId") String triggerId,
       @Body GoogleCloudBuildRepoSource repoSource);
+
+  @GET("/keel/{scmType}/{projectKey}/{repositorySlug}/manifests")
```

Comment:
commented on the igor pr, but also commenting here: I don't think these endpoints should be prefixed with `keel`. They should be descriptive of what they do not who they're for

Reply:
I struggled with naming for this. I thought of using `managed-delivery` but it occurred to me that the external feature name we use might change, whereas the service name was likely to stay forever, hence the current name.

Any suggestions for a better name? Perhaps remove `/keel` and just rename `/manifests` to `md-manifests` at the end?

Reply:
I agree with Emily - why we should limit the scope only for md/keel?

Reply:
To summarize our discussion in person:
- My concern was to avoid exposing an API that allows any caller on any Spinnaker instance the ability to  retrieve any file from any source control repo using igor's credentials. This was noted [here](https://github.com/spinnaker/igor/blob/6660b4b081ead1cdb69d5f76e00926a42eb8a82b/igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/KeelManifestsController.groovy#L58-L59).
- @emjburns's comments were more around exposing the name `keel` in the API, not so much restricting what source files the API exposes
- I will make changes to the API names as discussed but retain some level of restriction as to what kinds of directories and files can be accessed for the reason above